pkg/apimachinery/utils/verbs.go

```diff
@@ -19,4 +19,8 @@ const (
 	VerbDelete = "delete"
 	// VerbDelete is mapped from HTTP DELETE for collections
 	VerbDeleteCollection = "deletecollection"
+	// VerbGetPermissions is used when fetching resource specific permissions
+	VerbGetPermissions = "get_permissions"
+	// VerbUpdatePermissions is used when updating resource specific permissions
+	VerbUpdatePermissions = "update_permissions"
```

Comment:
```suggestion
	VerbManagePermissions = "manage_permissions"
```
or `set_permissions` -- we want something that implies create/update/delete without getting too verbose and avoid confustion

cc @chaudyg

Reply:
Changed to `set_permissions`